app.rb

```diff
@@ -1,7 +1,43 @@
 require 'sinatra/base'
+require './lib/player.rb'
+require './lib/game.rb'
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+
+  enable :sessions
+
+  before do
+    @game = Game.instance
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  post '/names' do
+    player = Player.new(params[:player_name])
+    @game = Game.create(player)
+    redirect('/play')
+  end
+
+  get '/play' do
+    erb :play
+  end
+
+  post '/rock' do
```

Comment:
You could potentially have a single endpoint for all the different moves, and this could come from a parameter in the request - in that case, you might want to double check and validate the move is valid before playing it